### DISCUSSION TITLE
Is there a way to use this universally in any situation outside of GW2? For example a .exe file?

### DISCUSSION THREAD

**Discussion by ACCESS-DENIIED** [Friendly0Fire/GW2Radial#282] [Ideas]
I absolutely love the ability to use a radial menu so that i do not have to look away from my screen to find awkward key locations.

Is there a way to use this menu universally and system wide in any application/game?

Or could you maybe suggest another program that would achieve this feature?

So far the only ones I have found are either awkward to use mid game (radial menu v4 - AHK) or do not centre in the middle of the screen when activated cause half of the menu to be inaccessible when turning the screen using action camera.

Thanks!

**Comment by Friendly0Fire** [comment]
That's an interesting idea! Unfortunately, you'd need to do fairly extensive modifications to this project to make it work standalone as an overlay. Anyone who wants is more than welcome to try and fork this as baseline if it helps, but it's not something I have the time to do unfortunately.
